### PR TITLE
Feature/tags meta header

### PR DESCRIPTION
**Pull request by domjngo**

### CODE REVIEW

File: author.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<div id="primary" class="content-area author-page">
+    <div class="container">
+        <div class="row">
+            <main id="main" class="col-xs-12 col-sm-8 col-md-8" role="main">
+                <article>
+                    <div class="entry-header">
+                        <h2>
+                            <?php
+                            $curauth = (isset($_GET['author_name'])) ? get_user_by('slug', $author_name) : get_userdata(intval($author));
```
Comment (netdesignr) [inline author.php:13]: Looking at this block of code I will possible think of:
1. Sanitize $_GET["author_name"] using something like e.g. sanitize_text_field($_GET["author_name"])
2. Adding it into the function and pass the $_GET variable as a parameter. By doing that you will end up having only functions inside this PHP file and unit test.

File: content-categories.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	</li>
+	<li>
+		<a href="#managing-information" role="tab" data-toggle="pill">Managing information</a>
+	</li>
+	<li>
+		<a href="#archives-and-archivists" role="tab" data-toggle="pill">Archives and archivists</a>
+	</li>
+</ul>
+<div class="tab-content clearfix">
+<?php foreach ( $categories as $cat ) {
```
Comment (netdesignr) [inline content-categories.php:23]: Possible the whole thing into a function? To stay away from inline PHP inside files wherever's possible?

File: content-latest.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?php
+$args = array(
```
Comment (netdesignr) [inline content-latest.php:2]: Maybe we can think of creating a generic Wp_query function where we could add options to it on the fly as parameters and reuse it whenever it's needed a WP_query if that makes sense? 
I'm happy to help, do not intend to give you extra work :). I believe we all could benefit from it in the future.

File: content.php [hunk 4]
```diff
@@ -12,16 +12,18 @@
 	<!-- singe.php -->
 	<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 		<div class="entry-header">
-			<h1><?php the_title(); ?></h1>
+			<h2><?php the_title(); ?></h2>
 		</div>
 		<div class="entry-meta">
+			<p>
```
Comment (netdesignr) [inline content.php:18]: Maybe can we wrap this into a function and try and reuse it? 
Sorry about my "function" obsession :)

**Review by netdesignr — CHANGES_REQUESTED** [review]

File: author.php (hunk 1 above)
Comment (domjngo) [reply author.php:13]: This is only checking if the $_GET exists. No need to sanitize it as it's not outputted in anyway.

File: content-categories.php (hunk 2 above)
Comment (domjngo) [reply content-categories.php:23]: Yes, we can at a later stage

File: content-latest.php (hunk 3 above)
Comment (domjngo) [reply content-latest.php:2]: Yes, this is a reusable template part. The old blog theme is a mess, but I can't steer too far away from the design. Later we can look at a more unified design, so we can have one Wp_query function to cover them all. So I have two reusable template parts that almost do the same thing. This is fine for now. When we revisit the Blog for the redesign then these can be addressed.

File: content.php (hunk 4 above)
Comment (domjngo) [reply content.php:18]: Yes, this is a reusable template part. This is fine for now. When we revisit the Blog for the redesign then these can be addressed then.

**Review by netdesignr — APPROVED** [review]

**Pull request merged**